### ISSUE TITLE
YouTube Screenshot Button clones

### ISSUE THREAD

**Issue by GrippenDynamik** [gurumukhi/youtube-screenshot#53]
Someone uploaded version 4.0.0 of your extension with different names:

1. https://addons.mozilla.org/en-US/firefox/addon/clip-shot/
2. https://addons.mozilla.org/en-US/firefox/addon/snapshot-tube/
3. https://addons.mozilla.org/en-US/firefox/addon/vid-snap/
4. https://addons.mozilla.org/en-US/firefox/addon/video-snapshot-master/
5. https://addons.mozilla.org/en-US/firefox/addon/video-screen-capture-pro/
6. https://addons.mozilla.org/en-US/firefox/addon/vidscreen/
7. https://addons.mozilla.org/en-US/firefox/addon/youtube-framegrab/
8. https://addons.mozilla.org/en-US/firefox/addon/youtube-frame-grabber-pro/
9. https://addons.mozilla.org/en-US/firefox/addon/youtube-stillify/
10. https://addons.mozilla.org/en-US/firefox/addon/youtube-video-capture-tool/
11. https://addons.mozilla.org/en-US/firefox/addon/youtube-video-still-maker/
12. https://addons.mozilla.org/en-US/firefox/addon/youtube-screengrab-pro/
13. https://addons.mozilla.org/en-US/firefox/addon/youtube-still-frame-master/
14. https://addons.mozilla.org/en-US/firefox/addon/video-frame-capture-toolkit/
15. https://addons.mozilla.org/en-US/firefox/addon/video-scene-snapshot-pro/
16. https://addons.mozilla.org/en-US/firefox/addon/video-thumbnail-maker-pro/
17. https://addons.mozilla.org/en-US/firefox/addon/youtube-video-still-maker/

If you report the users for abuse, Mozilla will probably remove the clones.

**Comment by joggee-fr** [comment]
Hi @GrippenDynamik, thanks for the report. Did you check the code of these extensions to know they are strict copies of this repo? If so, what is the goal? 
The sources of this repo are published using the MPL-2.0 license and nothing prevents someone to copy and distribute its own version. It may be a bit unfair but authorized.

**Comment by GrippenDynamik** [comment]
I've checked the manifest.json and README.md files.
The manifest.json in the clones and YouTube Screenshot Button are identical, except for the extension ID.
The README.md are 100% identical.

The goal is SEO spam.
Either the extension page's "Support site" link, or the user profile's "Homepage" link, is SEO spam.
(though a few of the extensions have neither; the spammer has either forgotten about them, or want to add the links later, because that's less suspicious)

I'm not trying to convince you that these clones need to be reported.
But this extension developer thought so, that's why I assumed other extension developers would feel the same way:
https://discourse.mozilla.org/t/someone-published-a-copy-of-my-add-on-on-amo/123606

**Comment by joggee-fr** [comment]
I have just checked the [first link](https://addons.mozilla.org/en-US/firefox/addon/clip-shot/) and you are 100% right. Code is exactly the same than version 4.0.0 of the repo. If the goal is SEO spam, i am OK to try reporting abuse to Mozilla. Did you already report it? Maybe you have already written a comprehensive message to be sent?

Note: I am not the owner of this repo nor the publisher of the extension on AMO, just user and contributor.

**Comment by GrippenDynamik** [comment]
I haven't reported any of the extensions listed in the OP, and I haven't composed a message to report them.

What I've done so far is report three unrelated clone extensions used for SEO spamming:
GetImageinfo pro plus
GetImageinfo plus
Gold Image info

Mozilla has deleted the first two.
The third one is still up, even though the Homepage and Support Site links are obvious SEO spam.

My reports to Mozilla contained only this information:
- "The extension <clone> is a copy of <original>"
- Link to the original
- List of SEO spam links on the extension page and/or user profile

**Comment by gurumukhi** [comment]
Thank you very much @GrippenDynamik for reporting this. 

I reported the first addon (and the developer account) using the 'report abuse' feature which you mentioned here originally. Now as you have put up the whole list here, it look more annoying. Let's just report all of these addons and developer accounts, hope Mozilla takes all these down.

>>  The sources of this repo are published using the MPL-2.0 license and nothing prevents someone to copy and distribute its own version. It may be a bit unfair but authorized.
@joggee-fr @GrippenDynamik what change do you suggest in the license here?

**Comment by gurumukhi** [comment]
Following is the message I am using for reporting these addons:

_This addon is exact copy of https://addons.mozilla.org/en-US/firefox/addon/youtube-screenshot-button/ (its open sourced code is stolen as it is and published as new addon here). 

Hence this addon abuses Content policy - "If the add-on is a fork of another add-on, the name must clearly distinguish it from the original and provide a significant difference in functionality and/or code."_

**Comment by joggee-fr** [comment]
@gurumukhi, I suggest absolutely no change in license. IMHO, the line in Mozilla addons policy is good enough and I support not too restrictive open-source licenses.

@GrippenDynamik, thanks again for finding such unfair copies. How did you find these addons? Is it a purely manual search?

I have checked all the provided links, all simple copies of version 4.0.0 and reported them to Mozilla.

**Comment by GrippenDynamik** [comment]
@gurumukhi 

I agree with @joggee-fr, this problem can't be solved with a license change.
If you want to allow legitimate forks (or mirrors), but forbid spam copies with only minor changes - that's impossible grasp with a license.

It should be enough to rely on Mozilla's terms or service, or Google's, if someone decides to put copies of your extensions there.
(I've seen spam copies of Chrome example extensions on AMO, so maybe there are spam copies of FF extensions in the CWS.)

@joggee-fr 

Finding new spam add-ons is mostly a manual process:
On AMO, press Enter in the "Find add-ons" field, then sort by "recently updated".
The spam extensions have no icon of their own (green puzzle piece), zero users, and long, pseudo-impressive names.

But when you've found one, you can usually find the copies.
You just have to use the right keyword, either on AMO or Google.

For https://discourse.mozilla.org/t/someone-published-a-copy-of-my-add-on-on-amo/123606 the keyword was "exif".

For the ones in the OP, the descriptions all contain "Shift+A".
Just found a new one: https://addons.mozilla.org/en-US/firefox/addon/youtube-video-still-maker/

**Comment by gurumukhi** [comment]
Wow, I just added 6 more addon links.
